integration_tests/e2e/assess/caseList.cy.ts

```diff
@@ -7,7 +7,6 @@ import type { Course, ReferralSummary } from '@accredited-programmes/models'
 
 const createReferralSummaries = (courseName: Course['name']): Array<ReferralSummary> => [
   referralSummaryFactory.build({ courseName, prisonNumber: 'ABC123', status: 'assessment_started' }),
-  referralSummaryFactory.build({ courseName, prisonNumber: 'ABC789', status: 'referral_started' }),
```

Comment:
Oops, after rebasing this with `main` (specifically [this change](https://github.com/ministryofjustice/hmpps-accredited-programmes-ui/pull/408/commits/dedfb1aa64407ff114aec9a0fb077c925fa79d88#diff-b5ec79a3153a624b7738a4e5404fe792563df5716ef55a9c4769ddaeb4aec11cR12-R21)), this might need updating. I think we could:
- add a trait to the referral summary factory, which picks a random status other than `referral_started` (this feels like `submitted`, but in the referral factory `submitted` currently means one specific status 🤔)
- extend `buildListWith` to take an optional trait
- pass in the trait to `builtListWith` here

What do you think?

Reply:
The approach I think you were taking just now (🦅) with the random status might also work, although invoking that function for each array item with `buildListWith` might be more of a challenge

Reply:
Opted for using `transientParams` in the end.  https://github.com/thoughtbot/fishery?tab=readme-ov-file#params-that-dont-map-to-the-result-object-transient-params